presto-main/src/main/java/com/facebook/presto/metadata/FunctionRegistry.java

```diff
@@ -576,7 +577,7 @@ public WindowFunctionSupplier load(SpecializedFunctionKey key)
                 .function(DECIMAL_SUM_AGGREGATION)
                 .function(DECIMAL_MOD_FUNCTION)
                 .functions(ARRAY_TRANSFORM_FUNCTION, ARRAY_REDUCE_FUNCTION)
-                .functions(MAP_FILTER_FUNCTION, MAP_TRANSFORM_KEY_FUNCTION, MAP_TRANSFORM_VALUE_FUNCTION)
+                .functions(MAP_FILTER_FUNCTION, MAP_TRANSFORM_KEY_FUNCTION, MAP_TRANSFORM_VALUE_FUNCTION, MAP_FROM_ENTRIES_FUNCTION)
```

Comment:
Let's move the registry align with `MAP_CONCAT_FUNCTION` and leave this line for lambdas only